sql/src/test/java/io/crate/analyze/TableDefinitions.java

```diff
@@ -142,6 +142,15 @@ public static Routing shardRoutingForReplicas(String tableName) {
                 add(null);
             }}).asIndexName())
         .build();
+    public static final TableIdent TEST_EMPTY_PARTITIONED_TABLE_IDENT =
+        new TableIdent(Schemas.DEFAULT_SCHEMA_NAME, "empty_parted");
+    public static final DocTableInfo TEST_EMPTY_PARTITIONED_TABLE_INFO = new TestingTableInfo.Builder(
+        TEST_EMPTY_PARTITIONED_TABLE_IDENT, new Routing(ImmutableMap.of()))
+        .add("id", DataTypes.INTEGER, null)
+        .add("name", DataTypes.STRING, null)
+        .add("date", DataTypes.TIMESTAMP, null, true)
+        .add("obj", DataTypes.OBJECT, null, ColumnPolicy.DYNAMIC)
```

Comment:
this column is not used anywhere, pls consider minimal setup always (to keep things as simple as possible)